src/Form/DataTransformer/PageDataTransformer.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ *
+ * This file is part of the Sonata for Ekino project.
+ *
+ * (c) 2018 - Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ */
+
+namespace Sonata\SonataHelpersBundle\Form\DataTransformer;
+
+use Sonata\PageBundle\Model\PageInterface;
+use Sonata\PageBundle\Model\PageManagerInterface;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Class PageDataTransformer.
+ *
+ * @author Benoit de Jacobet <benoit.de-jacobet@ekino.com>
+ */
+class PageDataTransformer implements DataTransformerInterface
```

Comment:
This class has the same behaviour as a normalizer.
It should be integrated using the serializer component of Symfony.
This way we could use context to handle different behaviours
instead of an internal `$keysToTransform` property